app/src/main/java/vn/mbm/phimp/me/newGallery.java

```diff
@@ -160,10 +163,17 @@
 
     // Local gallery
     static LinearLayout ln_local_gallery;
+    static LinearLayout localPhotosFrame;
+    static ScrollView localPhotosScroll;
+    static GridView localPhotosGrid;
+    static PhotosAdapter photosAdapter;
+    static Activity localActivity;
+    static Cursor pathcursor;
     static TextView txtlocal_gallery;
     static GridView gv_local_gallery;
     static ImageButton btn_local_more;
     static int local_rows_display = 0;
+    static LinearLayout ln_local_gallery;
```

Comment:
error: variable ln_local_gallery is already defined in class newGallery
See the line number-165